lib/mission_control/jobs/workers_relation.rb

```diff
@@ -18,6 +20,18 @@ def initialize(queue_adapter:)
     set_defaults
   end
 
+  # Returns a +MissionControl::Jobs::WorkersRelation+ with the configured filtering options.
+  #
+  # === Options
+  # * <tt>:hostname</tt> - To only include the workers of a given hostname.
+  # * <tt>:name</tt> - To only include the workers of a given name.
+  def where(hostname: nil, name: nil)
+    # Remove nil arguments to avoid overriding parameters when concatenating +where+ clauses
+    arguments = { hostname: hostname, name: name }.compact.collect { |key, value| [ key, value.to_s ] }.to_h
```

Comment:
I think I'd save the `to_s` on values here, just require callers to pass a string, so that this is simply
```suggestion
    arguments = { hostname: hostname, name: name }.compact
```

Reply:
I think it's ok not to be overly defensive here, so I wouldn't worry about it.